app/src/common/utils/permissions/permissions.js

```diff
@@ -36,6 +36,7 @@ export const canDeleteUser = checkPermission(ACTIONS.DELETE_USER);
 export const canSeeMembers = checkPermission(ACTIONS.SEE_MEMBERS);
 export const canEditOwnAccount = checkPermission(ACTIONS.EDIT_OWN_ACCOUNT);
 export const canDeleteLaunch = checkPermission(ACTIONS.DELETE_LAUNCH);
+export const canEditLaunch = checkPermission(ACTIONS.EDIT_LAUNCH); // requires userRole, projectRole, isOwner.
```

Comment:
What is the purpose of this comment?

Reply:
as i heard not all these check requires all 3 arguments. With a comment beside I'll know immediatly how many arguments I should pass for correct permission calculation.